packit_service/constants.py

```diff
@@ -287,3 +287,5 @@ def from_number(number: int):
     "grant `{packit_user}` user `Developer` permissions on the `{namespace}/{repo}`"
     " repository. You can do so [here]({url}/-/project_members)."
 )
+
+DISTGIT_URL = "https://src.fedoraproject.org"
```

Comment:
There's already a [`DISTGIT_URL` environment variable](https://github.com/packit/deployment/blob/main/openshift/packit-worker.yml.j2#L68) set to [`https://src.fedoraproject.org/` by default](https://github.com/packit/deployment/blob/main/playbooks/deploy.yml#L45).

There's also a [`PROD_DISTGIT_URL` defined in packit/constants.py](https://github.com/packit/packit/blob/main/packit/constants.py#L33)

I'd suggest using it like [here](https://github.com/packit/packit-service/blob/b147091ca383ac8ba884b8d647651abad200cc39/packit_service/worker/parser.py#L893)

Reply:
I haven't searched properly, thanks for the suggestion! I will change it